pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -228,7 +228,7 @@ in {
   zfsUnstable = common {
     # check the release notes for compatible kernels
     kernelCompatible = kernel.kernelOlder "5.19";
-    latestCompatibleLinuxPackages = linuxPackages_5_18;
```

Comment:
Hm, I never realised that this sort of thing could be a consequence of using zfs's latest compatible version. We should probably document that it may result in downgrades.

Reply:
I just opened https://github.com/openzfs/zfs/issues/13916. Hopefully this will get the ball rolling.